modules/tensor_mechanics/test/tests/smeared_cracking/cracking.cmp

```diff
@@ -0,0 +1,25 @@
+
+COORDINATES absolute 1.e-6    # min separation not calculated
+
+TIME STEPS relative 1.e-6 floor 0.0     # min:               0 @ t1 max:             0.1 @ t5
+
+
+# No GLOBAL VARIABLES
+
+NODAL VARIABLES relative 1.e-6 floor 1.e-4
+	disp_x  # min:               0 @ t1,n1	max:             0.1 @ t5,n5
+	disp_y  # min:               0 @ t1,n1	max:               0 @ t1,n1
+	disp_z  # min:               0 @ t1,n1	max:               0 @ t1,n1
+
+ELEMENT VARIABLES relative 1.e-6 floor 1.e-4
```

Comment:
It looks like you're going to need to increase the floor up to 5e-3 (which is fine because that's still a really small number for the stress).